CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -97,6 +107,8 @@ local OpenInventoryButton = nil
 local CloseInventoryButton = nil
 local InventoryFrame = nil
 local ScrollingFrame = nil
+local ScrollUpInventoryButton = nil
+local ScrollDownInventoryButton = nil
```

Comment:
It is a very interesting design, however, as you mentioned I thought it would be best to conform with the existing design. If we want to change it we probably want an entire overhaul later.